server/utils/match/index.ts

```diff
@@ -216,14 +216,17 @@ export const addressRow = (spaceSearchResult: SpaceSearchResult) => ({
   },
 })
 
-export const distanceRow = (spaceSearchResult: SpaceSearchResult, postcodeArea?: string) => ({
-  key: {
-    text: 'Distance',
-  },
-  value: {
-    text: `${spaceSearchResult.distanceInMiles} miles from ${postcodeArea || 'the desired location'}`,
-  },
-})
+export const distanceRow = (spaceSearchResult: SpaceSearchResult, postcodeArea?: string) => {
+  const roundedDistanceInMiles = Math.round(spaceSearchResult.distanceInMiles * 10) / 10
```

Comment:
I think this is fine but I guess the flipside is that a postcode area can probably cover a significant area itself, so I'm not completely sold on the value of the decimal point.

Reply:
true. I think if it came down to it being such a similar distance then the matcher would use other factors than distance to make the decision too